management/server/posture_checks.go

```diff
@@ -149,30 +151,20 @@ func (am *DefaultAccountManager) ListPostureChecks(ctx context.Context, accountI
 }
 
 // getPeerPostureChecks returns the posture checks applied for a given peer.
-func (am *DefaultAccountManager) getPeerPostureChecks(ctx context.Context, accountID string, peerID string) ([]*posture.Checks, error) {
+func (am *DefaultAccountManager) getPeerPostureChecks(ctx context.Context, account *Account, peerID string) ([]*posture.Checks, error) {
 	peerPostureChecks := make(map[string]*posture.Checks)
 
 	err := am.Store.ExecuteInTransaction(ctx, func(transaction Store) error {
```

Comment:
We can remove the store transaction here because we no longer interact with it

Reply:
yeah this I am removing